package.json

```diff
@@ -128,7 +128,8 @@
     "test": "lerna run test",
     "test:commit-check": "npm run doc:lint && npm run lint && npm run depcheck && npm run test",
     "test:security": "cd packages/udaru-hapi-server && npm run test:security",
-    "swagger-gen": "node scripts/getSwaggerJson.js > docs/swagger/swagger.json && swagger-gen docs/swagger/swagger.json -d docs/swagger && node scripts/injectUdaruSwaggerCss.js"
+    "swagger-gen": "node scripts/getSwaggerJson.js > docs/swagger/swagger.json && swagger-gen docs/swagger/swagger.json -d docs/swagger && node scripts/injectUdaruSwaggerCss.js",
+    "update-swagger": "node scripts/getSwaggerJson.js js > docs/swagger/swagger-json.js"
```

Comment:
where/when is this update-swagger task used now? Do we need to update the 'how to release' docs in contributing.md?

Reply:
It should probably replace swagger-gen, if swagger-gen is not replacing the script block then essentially it is not working, so suggest we replace the swagger-gen command with this command and remove existing?

Reply:
i think we should go with one approach or the other but not both.